src/View/User/Includes/_course_tab.html.twig

```diff
@@ -0,0 +1,90 @@
+{% block course_tab %}
+<div class="course-tab-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+    <div class="tab-title-text">
+        <p class="tab-title">
+            CONDITIONS
+        </p>
+        <p class="tab-text">
+            Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
+        </p>
+    </div>
+    <div class="tab-title-text">
+        <p class="tab-title">
+            INSCRIPTION
+        </p>
+        <p class="tab-text">
+            Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
+        </p>
+    </div>
+    <div class="btn-insc">
+        <button class="btn-inscription" type="button">
+            JE M'INSCRIS !
+        </button>
+    </div>
+    <p class="tab-title">
+        DETAILS
+    </p>
+    <div class="tab">
+        <button class="btn-course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse1">
+            Baby Poney
+        </button>    
+        <div class="collapse" id="collapse1">
+            <div class="card" style="width: 18rem;">
+                <img src="https://picsum.photos/id/1024/300/300" class="card-img-top" alt="Baby Poney Photo">
+                <div class="card-body">
+                    <p class="card-text">
+                    <p> Âge: à partir de 18 mois </p>
+                    <p> Horaires: </p>
+                    <p> Tarif: </p>
+                    </p>
```

Comment:
this looks like a list ;)

Reply:
I don't know why is that a problem, but i changed to <li> tags